celavi/pylca_celavi/pylca_opt_background.py

```diff
@@ -423,55 +121,68 @@ def runner(tech_matrix, F,i,l,j,k,route_id,final_demand_scaler):
         """
         tim0 = time.time()
         res = pd.DataFrame()
```

Comment:
Line 123 can be deleted - in the case that solver fails to find results, it should return an empty DataFrame